### PR TITLE
CDAP-14462 add docs for cdap env

### PR DESCRIPTION
**Pull request by yaojiefeng**
This pr depends on #10772 
https://builds.cask.co/browse/CDAP-DQB433-1

### CODE REVIEW

File: cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst [hunk 1]
```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
 Note that in the case of the Docker container running inside a Virtual Machine (as on
 either Mac OS X or Microsoft Windows), you will need to substitute the Docker VM's IP
 address for ``localhost`` in the web browser address bar.
+
+
+Customize JVM OPTS For Sandbox on Linux
```
Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: lower case `for` (or maybe just remove "For Sandbox on Linux")

File: cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst [hunk 2]
```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
...
+---------------------------------------
+On Linux, you can customize the jvm options for OutOfMemory error and whether to do a heapdump if an OutOfMemory error
```
Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: capitalize `JVM`

File: cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst [hunk 3]
```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
...
+happens. By default, CDAP process will be force killed if an OutOfMemory error happens and there will be no heapdump.
```
Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: s/happens/occurs/g (2 times)

Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: "By default, if an OutOfMemory error occurs, the CDAP Sandbox will be killed and no heap dump will be performed"

File: cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst [hunk 4]
```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
...
+You can configure these settings by editing the ``cdap-env.sh`` file under your ``conf`` directory. To change
```
Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: "To customize these settings, edit the ``cdap-env.sh`` file under the `./conf` directory" (./ added for consistency with `./bin` directory above)

File: cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst [hunk 5]
```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
...
+the behavior, change the value of ``KILL_ON_OOM_OPTS`` in ``cdap-env.sh``. To enable the heapdump, uncomment the line of
```
Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: "To enable the heap dump, ensure ``HEAPDUMP_ON_OOM`` is set to ``true`` in ``cdap-env.sh``" . (this is just to make the docs more resilient if we ever change that line in cdap-env.sh)

**Review by dereklwood — CHANGES_REQUESTED** [review]
some minor rewordings.

Maybe not for this PR, but I also think there should be a "Configuring Sandbox" section above this (prior to start/stop), where this should go.

**Comment by yaojiefeng** [comment]
Thanks for the review! I have opened https://issues.cask.co/browse/CDAP-14464 as an improvement to talk about the configuration we can do for sandbox.

File: cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst [hunk 6]
```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
 Note that in the case of the Docker container running inside a Virtual Machine (as on
 either Mac OS X or Microsoft Windows), you will need to substitute the Docker VM's IP
 address for ``localhost`` in the web browser address bar.
+
+
+Customize JVM OPTS
+------------------
+On Linux, you can customize the JVM options for OutOfMemory error and whether to do a heapdump if an OutOfMemory error
```
Comment (dereklwood) [inline cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst]: `heapdump` here vs `heap dump` elsewhere.  I *think* it's supposed to be `heap dump`

**Review by dereklwood — CHANGES_REQUESTED** [review]
one more minor inconsistency, otherwise lgtm

**Review by dereklwood — APPROVED** [review]

**Pull request merged**